misc/working_w_l10n_agency.md

```diff
@@ -0,0 +1,109 @@
+# Working with a Localization Agency
+
+This document is a guideline for project manager who interacts with a localization agency that is already vetted.
+
+## Initial set up before sending localization requests
+
+### Ask for a rate sheet
+
+* Make sure you have received a rate sheet customized for Mozilla needs. This should have been sent as part of the vetting process, and it can be updated annually. An example of the official [rate sheet](https://drive.google.com/a/mozilla.com/file/d/0B7u_NzvpgGcgTDhoVFBsN3dMc0E/view?usp=sharing) that includes the following but not limited to:
+  * Word rate for each of the interested locales as in new words, 100% match, fuzzy match for translation, review, proofreading
+  * Hourly rate: engineering, QA and testing, PM rate.
+  * Minimum charge for a project and the thresholds by number of words per locale.
+  * Rush rate.
```

Comment:
Nope, but it’s a good point. Changing it into something like `Rush rate, and its threshold.` Is this proper English?